src/components/Nav.js

```diff
@@ -33,23 +46,17 @@ export default function Nav() {
             {links.map((link, index) => (
               <li key={index}>
                 <a href={link.href} rel='noopener noreferrer' target='_blank'>
-                  {link.text}
+                  <span>
+                    {link.text}{' '}
+                    <span className='sr-only'>opens in new window</span>
```

Comment:
So we would not need to display these icons when the navigation items are not collapsed.
I understand the confusion as the usage is not documented yet.

Reply:
Sorry, I'm not sure I understand this comment. When the menu is collapsed then the `<ul>` is hidden and nothing in the menu is showing.

Reply:
Here is the discussion if you would like to take a look: https://github.com/freeCodeCamp/freeCodeCamp/issues/41373
Ultimately we would need to make the navigation vertical on coderadio for consistency and then add the icons only when they are collapsed; For now, we could remove all icons for simplicity.

Reply:
Thanks for the link to the issue. To be honest, I didn't quite find a good explanation there but that could just be me. Regardless, the icon is not technically a WCAG requirement (but it is considered a best practice) so if you don't want it there then I'll remove it.